### DISCUSSION TITLE
Add more examples!

### DISCUSSION THREAD

**Discussion by zirnc** [microsoft/WindowsAppSDK#1318] [Ideas] [answered]
Now there are 5 examples in [WindowsAppSDK-Samples](https://github.com/microsoft/WindowsAppSDK-Samples). They are really helpful, but I think MS can add more examples :joy: such as notepad, image viewer and so on..

**Comment by btueffers** [comment, marked as answer]
@ChungZH We'll be adding more samples as the Windows App SDK evolves, thank you for the feedback!

**Reply by zirnc** [reply]
Yes, as you can see, not many people use WinUI 3.0 now. So the official example is very important.

**Reply by float34** [reply]
@btueffers Sorry, this is not enough. We need more examples of the current functionality. Even when you have, say, only one feature in your framework you absolutely need to have several examples which demonstrate its usage. Especially when you have a new evolving platform.
We don't even have an access to the source code - if it was the case we would figure it out ourselves, of course. But we rely on you.

**Reply by btueffers** [reply]
Hi @torquerrr, do you have specific examples of what samples you would expect to see and aren't available? We'd like to better understand what would benefit your scenarios. The WindowsAppSDK-Samples repo includes samples for the 1.0 features and the ReadMe links to additional samples resources such as the [Xaml Controls Gallery](https://github.com/microsoft/Xaml-Controls-Gallery) and the [Windows Composition Gallery](https://github.com/microsoft/WindowsCompositionSamples). The Docs also include code snippets for specific features where applicable.
